README.md

```diff
@@ -11,3 +11,47 @@ Add following step to your workflow:
     delete_release: true #(optional) default: true 
     repo: <owner>/<repoName> #(optional) target repository. default: repo running this action
 ```
+
+## Developing
+
+You can follow these instructions to get this project cloned to your local machine and ready for development. Note that
+development requires at least [Node 16](https://nodejs.org/en/download) to be installed on your computer.
+
+```shell
+git clone https://github.com/dev-drprasad/delete-tag-and-release.git
+cd delete-tag-and-release
+npm install
+```
+
+Tests can be run via:
+
+```shell
+npm test
+```
+
+Since this is a repo for a GitHub Action, **every new commit that changes code must include a new bundle.** You can 
+generate a new bundle via running:
+
+```shell
+npm run package
+```
+
+and commiting the resulting changes that will be in the `dist/` folder.
+
+## Releasing
+As this repo is a GitHub Action, releases are done via creating a new tag and pushing that tag to the repo. A new
```

Comment:
```suggestion
Releases are done by creating a new tag and pushing that tag to the repo. A new
```
Let's try to use Simple English, to make this easily translatable for those that don't know English natively.